manifests/init.pp

```diff
@@ -197,39 +221,40 @@
 #   Whether the module should try to manage the pcsd service in addition to the
 #   corosync service.
 #   pcsd service is the GUI and the remote configuration interface.
-#   Defaults to false
+#   Default: false
 #
 # [*cluster_name*]
 #   This specifies the name of cluster and it's used for automatic
 #   generating of multicast address.
+#   Default: undef
 #
 # [*join*]
 #   This timeout specifies in milliseconds how long to wait for join messages
 #   in the membership protocol.
-#   Default to 50
+#   Default: undef
 #
 # [*consensus*]
 #   This timeout specifies in milliseconds how long to wait for consensus to be
 #   achieved before starting a new round of membership configuration.
 #   The minimum value for consensus must be 1.2 * token. This value will be
 #   automatically calculated at 1.2 * token if the user doesn't specify a
 #   consensus value.
-#   Defaults to false,
+#   Default: undef
 #
 # [*max_messages*]
 #   This constant specifies the maximum number of messages that may be sent by
 #   one processor on receipt of the token. The max_messages parameter is limited
 #   to 256000 / netmtu to prevent overflow of the kernel transmit buffers.
-#   Defaults to 17
+#   Default: undef
```

Comment:
If 17 is to remain there needs be a comment stating that this the default value according to corosync.conf(5)

Reply:
Thanks for all the feedback. I documented values defined in `params.pp` as the defaults; e.g. `max_messages` is `undef` since 2016-09-05. So actually `17` is long gone.